ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -182,6 +200,24 @@ Item {
         id: transactionBubble
         TransactionBubble {}
     }
+
+    // Image Modal
+    ModalPopup {
```

Comment:
Could you put the ModalPopup in the ChatColumn.qml next to the ProfilePopup. That way they are at the same place and also if you put the Popup inside `Message`,  it means we initialize a new Popup for each Message

Reply:
I've created a new `ImagePopup` file placed the component next `ProfilePopup` in `ChatColumn.qml`. I've also binded an `imageClick` function to `imagePopup.openPopup`. Geez, I'll be more considerate memory in future.